grapher/lineCharts/LineChartUtils.ts

```diff
@@ -10,12 +10,20 @@ export const columnToLineChartSeriesArray = (
     const { isProjection, owidRowsByEntityName } = col
     const entityNames = Array.from(owidRowsByEntityName.keys())
     return entityNames.map((entityName) => {
-        const seriesName =
-            seriesStrategy === SeriesStrategy.entity
-                ? entityName
-                : canSelectMultipleEntities
-                ? `${entityName} - ${col.displayName}`
-                : col.displayName
+        let seriesName = ""
```

Comment:
nit: if you use `let seriesName: string` without assigning it, TypeScript will check that the variable is being assigned in all branches

Reply:
Ah nice, I didn't know this - ML languages don't let you do this :)